libs/top_level_metrics.py

```diff
@@ -99,17 +94,15 @@ def calculate_metrics_for_timeseries(
         cumulative_cases, data[CommonFields.CONTACT_TRACERS_COUNT]
     )
     top_level_metrics_data = {
-        CommonFields.DATE: data[CommonFields.DATE],
         CommonFields.FIPS: fips,
         MetricsFields.CASE_DENSITY_RATIO: case_density,
```

Comment:
what's the relationship between MetricsFields, can_model_output_schema and can_api_definition?

Reply:
can_model_output_schema is pretty independent from the others, it's just some very old code around what fields are in the webui output files. 

`MetricsFields` are loosely the metrics in the output data frame from `calculate_metrics_for_timeseries`.  The values of them match the keys of the `Metrics` object in can api definition.  

In general I'm still a bit unsure of how to deal with data frames like this.  A lot of time, i'd like similar functionality from the TimeseriesDataset. I definitely want to make sure we're not referring to columns by strings defined in various places. but I haven't found anything I love so far.  For instance, I thought about making a `class MetricsTimeseries(TimeseriesDataset)` but it didn't quite feel right.